data/mainnet/tokens.js

```diff
@@ -5546,4 +5546,106 @@ module.exports = {
     ],
     cmcRewardTokenSymbols: ['iFARM', 'DAI', 'WETH'],
   },
+  bal_TUSD_STABLE: {
```

Comment:
This vault earns Matic and TUSD on top of the BAL. These need to be added too. The `BALANCER` apy-function might need an update to include the 3rd reward token.